src/database.js

```diff
@@ -0,0 +1,33 @@
+import { MongoClient } from 'mongodb';
+
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { DB_USER, DB_PASSWORD } = process.env;
+
+const MONGO_URL = `mongodb://${DB_USER}:${DB_PASSWORD}@ds227171.mlab.com:27171/theremotants`;
+const DB_NAME = 'theremotants';
+
+let connectionInstance;
+
+const getDbClient = async () => {
+  if (!connectionInstance) {
```

Comment:
@Saumya29 not to use bang operator in javascript, because it makes truth to both null and undefined values. Always prefer `===`, like `connectionInstance === null`.  and also only checking  whether variable is null check its server status too https://docs.mongodb.com/manual/reference/command/serverStatus/